.github/workflows/changelog.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: add-no-changelog-label
     steps:
-    - uses: scientific-python/action-towncrier-changelog@v1
+    - uses: scientific-python/action-towncrier-changelog@v1 # zizmor: ignore[unpinned-uses]
```

Comment:
The `unpinned-uses` rule checks that actions used via `uses` are pinned to a specific commit rather than the version.  We can ignore this check for actions that come from trusted publishers.